### DISCUSSION TITLE
Add Make.com (integromat) integration please.

### DISCUSSION THREAD

**Discussion by premiumlion** [deepgram/community#152] [Feature Requests]
Hi, at the moment, I've only found Deepgram on Pipedream.

Would love to see it on make.com

Thank you.

**Comment by nosepoke** [comment]
Same! I've been trying to connect Deepgram using the HTTP "make a request" module, but thus far I haven't been able to get it to work.

**Comment by jpvajda** [comment]
@premiumlion  @nosepoke  👋  Thanks for asking about Make + Deepgram. We are exploring  this currently.

**Reply by premiumlion** [reply]
Hi, any progress on this? Thank you.

**Comment by jpvajda** [comment]
@premiumlion 👋  We are actually working on a Custom Make App currently and it's in review waiting on approval with Make.  I'll add @SandraRodgers to this discussion as she is currently working on this Custom App.

**Comment by SandraRodgers** [comment]
@premiumlion I'll share the custom app with you. It's still in review, but with this link you can try it out. Let me know your experience with it and if you run into problems. Once it's added to make.com as an official integration, I'll be writing documentation to support using it in various scenarios. 

https://www.make.com/en/hq/app-invitation/127b90571f7c3c033b9c9de38c8ca6b2

**Comment by premiumlion** [comment]
Amazing, thank you. I will test this out when I have a moment.
